tfhe/src/high_level_api/integers/tests.rs

```diff
@@ -271,11 +271,83 @@ fn fhe_uint32_shift(config: Config) {
 }
 
 #[test]
-fn test_uint32_shift() {
+fn test_uint32_bitwise() {
```

Comment:
no more shifts ?

Reply:
They still exist above, the diff is a bit misleading

Reply:
![image](https://github.com/zama-ai/tfhe-rs/assets/15125341/4d3fc1fa-2ce2-4432-8525-f08b2a71f67a)

Reply:
are the tests still launched ? I'm a bit tired to be honest I have a hard time seeing if it's the case

Reply:
looks to me they aren't launched but I could be wrong